### ISSUE TITLE
Error when InfoBox is displayed

### ISSUE THREAD

**Issue by AdamFinley** [AnalyticalGraphicsInc/cesium#2545]
To reproduce:
1. Go to 1.7 sandcastle
2. Open Web Map Service (WMS) example 
3. Click in data region

Rendering stops and error is displayed.  Console output:
```
An error occurred while rendering.  Rendering has stopped.
undefined
TypeError: Cannot read property 'background-color' of null
TypeError: Cannot read property 'background-color' of null
    at null.wa (http://cesiumjs.org/Cesium/Source/Widgets/InfoBox/InfoBox.js:123:63)
    at Function.A.notifySubscribers (http://cesiumjs.org/Cesium/Source/ThirdParty/knockout-3.2.0.js:33:55)
    at Function.d.W (http://cesiumjs.org/Cesium/Source/ThirdParty/knockout-3.2.0.js:36:179)
    at d [as _descriptionSanitizedHtml] (http://cesiumjs.org/Cesium/Source/ThirdParty/knockout-3.2.0.js:36:71)
    at knockout.defineProperty.set (http://cesiumjs.org/Cesium/Source/Widgets/InfoBox/InfoBoxViewModel.js:100:56)
    at f [as description] (http://cesiumjs.org/Cesium/Source/ThirdParty/knockout-3.2.0.js:43:512)
    at Viewer._onTick (http://cesiumjs.org/Cesium/Source/Widgets/Viewer/Viewer.js:1325:46)
    at Event.raiseEvent (http://cesiumjs.org/Cesium/Source/Core/Event.js:146:30)
    at Clock.tick (http://cesiumjs.org/Cesium/Source/Core/Clock.js:203:21)
    at CesiumWidget.render (http://cesiumjs.org/Cesium/Source/Widgets/CesiumWidget/CesiumWidget.js:638:39)
```

Can also trigger error by making a simple entity with a label and description and clicking it to open InfoBox.

**Comment by mramato** [comment]
Thanks, this was definitely a big good on our end.  I'm looking into the possibility of doing a Cesium 1.7.1 now.

**Comment by pjcozzi** [comment]
Thanks again for the report.  This was fixed in #2554.

**Comment by AdamFinley** [comment]
Very impressed with the fast response and resolution of this issue!  Thank you!
